system/Database/SQLSRV/Builder.php

```diff
@@ -295,10 +295,15 @@ private function getFullName(string $table): string
                 $database = $this->db->getDatabase();
                 $table    = $dbInfo[0];
 
-                if (isset($dbInfo[2])) {
-                    $database  = str_replace('"', '', $dbInfo[0]);
-                    $schema    = str_replace('"', '', $dbInfo[1]);
-                    $tableName = str_replace('"', '', $dbInfo[2]);
+                if (count($dbInfo) >= 2) {
+                    if (count($dbInfo) === 3) {
+                        $database  = str_replace('"', '', $dbInfo[0]);
+                        $schema    = str_replace('"', '', $dbInfo[1]);
+                        $tableName = str_replace('"', '', $dbInfo[2]);
+                    } else {
+                        $schema    = str_replace('"', '', $dbInfo[0]);
+                        $tableName = str_replace('"', '', $dbInfo[1]);
+                    }
```

Comment:
it always at least 2, so direct check 3 should be ok:

```php
                if (count($dbInfo) === 3) {
                    $database  = str_replace('"', '', $dbInfo[0]);
                    $schema    = str_replace('"', '', $dbInfo[1]);
                    $tableName = str_replace('"', '', $dbInfo[2]);
                } else {
                    $schema    = str_replace('"', '', $dbInfo[0]);
                    $tableName = str_replace('"', '', $dbInfo[1]);
                }
```

Reply:
Done.